test/e2e/proxy_test.go

```diff
@@ -773,7 +774,7 @@ func TestSpaceLister(t *testing.T) {
 
 			// then
 			require.NoError(t, err)
-			verifyHasExpectedWorkspace(t, expectedWorkspaceFor(t, awaitilities.Host(), users["bicycle"], true), *bicycleWS)
+			verifyHasExpectedWorkspace(t, expectedWorkspaceFor(t, awaitilities.Host(), users["bicycle"], true, appStudioTierRolesWSOption), *bicycleWS)
```

Comment:
it's just a detail, but since you introduced the options, then you could update the function to:
```go
func expectedWorkspaceFor(t *testing.T, hostAwait *wait.HostAwaitility, user *proxyUser, additionalWSOptions ...commonproxy.WorkspaceOption) toolchainv1alpha1.Workspace {
```
and then call it:
```suggestion
			verifyHasExpectedWorkspace(t, expectedWorkspaceFor(t, awaitilities.Host(), users["bicycle"], commonproxy.WithType("home"), appStudioTierRolesWSOption), *bicycleWS)
```

Reply:
Good point! Thanks!

I've replaced the parameter with the option. Please see: https://github.com/codeready-toolchain/toolchain-e2e/pull/797/commits/2fb797da4667ef5afac1f6fb601caeef7b75ad42